src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -234,6 +235,7 @@ public ServicePoint getCheckoutServicePoint() {
 
   public Loan renew(DateTime dueDate, String basedUponLoanPolicyId) {
     changeAction("renewed");
+    changeActionComment(null);
```

Comment:
I might be tempted to extract a new method that hides this detail, called `removeActionComment`

Reply:
Done